src/css/style.scss

```diff
@@ -11,6 +11,11 @@
   text-align: right;
 }
 
+.usa-table tfoot td {
+  font-weight: bold;
+  background-color: #dedede;
+}
```

Comment:
As far as I can tell... USWDS does not style `tfoot` at all... I might consider opening a PR that just copies the `thead` ones? 🤷